velox/docs/functions/spark/aggregate.rst

```diff
@@ -11,6 +11,37 @@ General Aggregate Functions
 
     Returns the bitwise XOR of all non-null input values, or null if none.
 
+.. spark:function:: bloom_filter_agg(hash, estimatedNumItems, numBits) -> varbinary
+
+    Creates bloom filter from input hashes and returns it serialized into VARBINARY.
+    The caller is expected to apply xxhash64 function to input data before calling bloom_filter_agg.
+    For example, 
+        bloom_filter_agg(xxhash64(x), 100, 1024)   
+    In Spark implementation, ``estimatedNumItems`` and ``numBits`` are used to decide the number of hash functions and bloom filter capacity.
+    In Velox implementation, ``estimatedNumItems`` is not used.
+
+    ``hash`` cannot be null.
+    ``numBits`` specifies max capacity of the bloom filter, which allows to trade accuracy for memory.
+    In Spark,  the value of``numBits`` is automatically capped at 67,108,864.
+    In Velxo, the value of``numBits`` is automatically capped at 716,800.
```

Comment:
typo: Velxo -> Velox

> is automatically capped at 716,800.

Let's update PR description to explain where this limitation comes from. CC: @xiaoxmeng